App/components/Bible/Verse.js

```diff
@@ -1,24 +1,80 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
 import styled from '@emotion/native';
 import { useSelector } from 'react-redux';
-import PropTypes from 'prop-types';
 import { selectors } from 'store';
+import { useIsMounted } from 'lib/hooks';
 
 export default function Verse({ number, text }) {
-  // TODO: Account for italics in english
-  const fontSize = useSelector(selectors.fontSize);
+  const size = useSelector(selectors.fontSize);
+  const isEnglish = useSelector(selectors.language) === 'english';
+
+  if (!isEnglish)
+    return (
+      <VerseContainer size={size}>
+        <Text size={size}>
+          {number} {text}
+        </Text>
+      </VerseContainer>
+    );
+
+  const isMounted = useIsMounted();
+
+  const breakVerseIntoParts = () => text.split(/\[|\]/);
+
+  const doesItStartWithItalics = () => text.startsWith('[');
+
+  const isItalicized = (i) => i % 2 === (startsWithItalics ? 0 : 1);
```

Comment:
It would probably be a bit more clear if `startsWithItalics` and `setStartsWithItalics` were not referenced in the code until after they are defined. Personally I'd move lines 35 and 36 above this section, but that's just an opinion.

Reply:
I changed those from arrow to regular functions (hoisting issues) and put them down below

Reply:
Sounds good!